bin/offline-helm.sh

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env bash
 
 set -euo pipefail
+set -x
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+ANSIBLE_DIR="$( cd "$SCRIPT_DIR/../ansible" && pwd )"
+ansible-playbook -i "$ANSIBLE_DIR"/inventory/offline "$ANSIBLE_DIR"/helm_external.yml --skip-tags=rabbitmq-external -vv
```

Comment:
And, why do we need `-vv` while other playbooks are run without this level of verbosity?

Reply:
makes sense, removing from here as well.